src/common/simple-manifest.js

```diff
@@ -0,0 +1,31 @@
+const getSimpleManifest = function (manifest) {
+  // This function generates a simplified manifest with the public and private
+  // variable fields removed. All public and private variables are defined in the
+  // environment field of each container
+
+  const simpleManifest = { ...manifest['manifest'] }
+  delete simpleManifest['vars']
+  const containers = manifest['manifest']['containers']
+  const publicVars = manifest['manifest']['vars']
+  const privateVars = manifest['private'] ? manifest['private']['vars'] : {}
+
+  simpleManifest['containers'] = containers.reduce((agg, container) => {
+    const env = container['environment']
+    const newEnv = {
+      ...env,
+      ...publicVars,
+      ...privateVars
+    }
+    const keys = Object.keys(newEnv)
+    if (keys.length) {
+      agg.push(newEnv)
```

Comment:
seems like this will drop all the fields of the container aside from the environment